src/main/java/io/netty/incubator/codec/http3/Http3ClientConnectionHandler.java

```diff
@@ -21,13 +21,27 @@
 
 import java.util.function.LongFunction;
 
+import static io.netty.incubator.codec.http3.Http3SettingsFrame.HTTP3_SETTINGS_QPACK_MAX_TABLE_CAPACITY;
+
 public final class Http3ClientConnectionHandler extends Http3ConnectionHandler {
 
+    private final LongFunction<ChannelHandler> pushStreamHandlerFactory;
+
     /**
      * Create a new instance.
      */
     public Http3ClientConnectionHandler() {
-        this(null, null, null, false);
+        this(null);
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * @param pushStreamHandlerFactory the {@link ChannelHandler} which will be notified about
+     * {@link Http3RequestStreamFrame}s or {@code null} if the user is not interested in these.
```

Comment:
The javadocs seems to be wrong... Just copy it from below and you should be good. Also I wonder if we really need this extra constructor